server/src/schema/event/eventType.ts

```diff
@@ -38,8 +37,20 @@ export const eventType = builder.prismaObject('TumiEvent', {
     registrationLink: t.exposeString('registrationLink', { nullable: true }),
     registrationMode: t.expose('registrationMode', { type: RegistrationMode }),
     participantText: t.exposeString('participantText'),
-    organizerText: t.exposeString('organizerText'),
+    organizerText: t.string({
```

Comment:
this should be solved with the scope auth and a resolver for unauthenticated access